src/components/TodoHeader/TodoHeader.tsx

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable jsx-a11y/control-has-associated-label */
+import { useState } from 'react';
+import { Todo } from '../../types/Todo';
+import { ResponseError } from '../../types/enum';
+
+type Props = {
+  todos: Todo[];
+  toggleTodosActive: () => void;
+  setIsShowFooter: (arg: boolean) => void;
+  setError: (arg: ResponseError) => void;
+  isDisable: boolean;
+  setIsLoading: (arg: boolean) => void;
+  setCreatingTodoTitle: (arg: string) => void;
+  headerAddTodo: (arg: string) => void;
+};
+
+export const TodoHeader: React.FC<Props> = ({
+  todos,
+  toggleTodosActive,
+  setError,
+  isDisable,
+  setIsLoading,
+  setCreatingTodoTitle,
+  headerAddTodo,
+}) => {
+  const [todoInput, setTodoInput] = useState('');
+
+  const todoFormHandler = (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    setIsLoading(true);
+
+    if (!todoInput.trim()) {
+      setIsLoading(false);
+
+      return setError(ResponseError.EMPTY);
+    }
+
```

Comment:
```suggestion
   if (!todoInput.trim()) {
      setIsLoading(false);

      return setError(ResponseError.EMPTY);
    }

    setIsLoading(true);
```

to avoid redundant setIsLoading invokes